daml-lf/archive/da/daml_lf_1.proto

```diff
@@ -28,12 +28,12 @@
 // * 5 -- 2019-05-22: Relax serializability constraints for contract ids
 //        2019-05-23: Add BuiltinFunction.COERCE_CONTRACT_ID
 //        2019-05-24: Make actors in exercise optional
-// * dev (special staging area for the next version to be released)
-//        2019-05-27: Add enum type.
+// * 6 -- 2019-05-27: Add enum type.
 //        2019-06-04: Add BuiltinFunction.{TEXT_FROM_CODE_POINTS, TEXT_TO_CODE_POINTS}
 //        2019-06-12: Add Package.interned_package_ids and PackageRef.interned_id
 //        2019-07-04: Transaction submitters must be in contract key maintainers when looking up. See #1866.
-
+// * dev (special staging area for the next version to be released)
+//        2019-07-01: no change yet.
```

Comment:
Should this be changed to a later date than the latest date in 1.6?

Reply:
fixed